Formula/sip.rb

```diff
@@ -32,18 +33,30 @@ class Sip < Formula
     sha256 "b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f"
   end
 
+  # TODO: remove them after sip 6.1.0
+  # These patch provide the option `--scripts-dir`
+  patch do
+    url "https://www.riverbankcomputing.com/hg/sip/raw-diff/ffd0551c32cc/sipbuild/builder.py"
+    sha256 "2c969dfba2e4b0553d06999a3aa07a93ea4b7ca2cce62635d1418ecdc74a6df2"
+  end
+
+  patch do
+    url "https://www.riverbankcomputing.com/hg/sip/raw-diff/ffd0551c32cc/sipbuild/project.py"
+    sha256 "ea99834ab404583a1a49a05997950758fe95ba473129438117ffa5647028d99a"
+  end
+
   def install
-    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
-    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
-    %w[packaging pyparsing toml].each do |r|
-      venv.pip_install resource(r)
+    python = Formula["python@3.9"]
+    venv = virtualenv_create(libexec, python.bin/"python3")
+    resources.each do |r|
+      venv.pip_install r
     end
 
-    system Formula["python@3.9"].opt_bin/"python3", *Language::Python.setup_install_args(prefix)
+    system python.bin/"python3", *Language::Python.setup_install_args(prefix)
```

Comment:
Actually, now that I think about this, why can't we just do a `virtualenv_install_with_resources` here?

Reply:
Even after writing the `homebrew-sip.pth` file? Weird.

Reply:
Because `sip` is both python library and Python application, but (document)[https://docs.brew.sh/Python-for-Formula-Authors] doesn't mention this situation